<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDirectCamView</class>
 <widget class="QDockWidget" name="CDirectCamView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>608</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>472</width>
    <height>371</height>
   </size>
  </property>
  <property name="styleSheet">
   <string notr="true">QDockWidget::title {
	background-color: rgb(174, 174, 220);
	color: rgb(45, 45, 45);
	 border: 1px solid black;
     border-top: 3px solid qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 black, stop: 1  rgb(174, 174, 220));
     border-radius: 2px;
 }

QPushButton {
     
	color: rgb(45, 45, 45);
     border: 2px solid #8f8f91;
	 border-color: rgb(90, 93, 125);
     border-radius: 6px;
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #f6f7fa, stop: 1 #dadbde);
     min-width: 80px;
 }

 QPushButton:pressed {
     background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                       stop: 0 #dadbde, stop: 1 #f6f7fa);
 }

 QPushButton:hover {
     border-color: rgb(90, 93, 255);
 }

 QPushButton:default {
     border-color: navy; /* make the default button prominent */
 }

QPushButton:disabled {  
	color: rgb(171, 171, 171);
     border-color: none; /* make the default button prominent */
 }</string>
  </property>
  <property name="windowTitle">
   <string>Direct Cam View</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="CDirectCamGraphicsView" name="m_graphicsView">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="CRulerWidget" name="m_horRuler_wdgt" native="true">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="CRulerWidget" name="m_verRuler_wdgt" native="true">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>50</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSlider" name="m_frameSlider">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="m_firstFrame_btn">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_prevFrame_btn">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_playPause_btn">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_nextFrame_btn">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_lastFrame_btn">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="m_startStop_btn">
          <property name="text">
           <string>Start Streaming</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Select Cam Resolution:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="m_camRes_comBx"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QPushButton" name="m_openFile_btn">
          <property name="text">
           <string>Open File</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_saveImage_btn">
          <property name="text">
           <string>Save Image</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="m_startStopRec_btn">
      <property name="text">
       <string>Start Recording</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="m_clr_btn">
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CDirectCamGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>..\include\ui\directcam_graphicsview.hpp</header>
  </customwidget>
  <customwidget>
   <class>CRulerWidget</class>
   <extends>QWidget</extends>
   <header>..\include\ui\ruler_widget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
